x-pack/legacy/plugins/uptime/public/hooks/use_telemetry.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useEffect } from 'react';
+import { HttpHandler } from 'kibana/public';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+
+export enum UptimePage {
+  Overview = 0,
+  Monitor,
+  NotFound,
+}
+
+const getApiPath = (page: UptimePage) => {
+  switch (page) {
```

Comment:
WDYT about using string enums with the URLs as values? That way we wouldn't need to have a case per page. The enum would actually be the path value. You could just check that the value is not nil and not the 404 value and return the string.

Reply:
I like it, good idea. See https://github.com/elastic/kibana/pull/53726/commits/cff6ea650ae71eef7e9915749e27adfef68b7d19 for implementation.